src/main.py

```diff
@@ -22,6 +24,14 @@ def __init__(self) -> None:
         self.end_commands = serialize.get_end_commands()
         self.config = serialize.get_config()
 
+        self.command_generator = command_generator.CommandGenerator()
+        # Create instances of the generate_* methods
```

Comment:
Instead of copying the method reference, modify the code in other places to use the `self.command_generator` instance.